docs/reference/workflows/overview.md

```diff
@@ -0,0 +1,29 @@
+# Overview
+
+The SpatialOS GDK for Unity provides functionality to develop games in both of Unity's paradigms:
+
+* Object-oriented: MonoBehaviours and GameObjects
+* Data-oriented: Entity-Component-System (ECS)
+
+You can stick to familiar MonoBehaviour-based development, check out the new ECS workflow, or try a combination of both.
+
+> However you choose to develop your game, it is important to understand what [SpatialOS entities]({{urlRoot}}/reference/glossary#spatialos-entity) are and how they relate to each workflow.
+
+## MonoBehaviour-centric workflow
+
+In the GDK, SpatialOS entities can be linked to GameObjects. [Readers and Writers]({{urlRoot}}/reference/workflows/monobehaviour/interaction/reader-writers/index) allow you to inspect and change the state of components on SpatialOS entities using MonoBehaviours that you add to their linked GameObjects.
+
+You may find that not all SpatialOS entities need to be represented as a GameObjects. The [Game Object Creation Feature Module]({{urlRoot}}/modules/game-object-creation/overview) allows you to customise the GameObject creation and linking process.
+
+## ECS-centric workflow
+
+ECS is a new Unity development paradigm, introduced by Unity in early 2018.
+
+In the GDK, each SpatialOS entity checked out by a worker has a corresponding ECS entity. To inspect or change the state of components on SpatialOS entities, you must read or write to the components on corresponding ECS entities.
+
+To learn more about ECS, check out:
```

Comment:
not for this PR, but we really need some kind of introduction page on ECS. We don't have a single link here on how to get started with our ECS-centric workflow

Reply:
I agree. Part of the plan is to have a similar overview page for MBs/ECS which then lays the ground work to link to worker entities / reader/writers / components / reactive vs nonreactive etc